### ISSUE TITLE
Set number of tuners

### ISSUE THREAD

**Issue by Th0masDB** [morphis/tvhproxy-snap#2]
In plex it says at the moment that I have 6 tuners, but I only have 2 tuners. 
How can I change this?

![image](https://user-images.githubusercontent.com/12988955/110244420-e58fec00-7f5e-11eb-8daa-80c7cf7815e3.png)

**Comment by amsound** [comment]
Yeah I'd like to be able to adjust this - all of the snap files are read-only so haven't been able to adjust manually.
